sqlmesh/core/audit/definition.py

```diff
@@ -240,7 +240,7 @@ def macro_definitions(self) -> t.List[d.MacroDef]:
     def _create_query_renderer(self, model: Model) -> QueryRenderer:
         return QueryRenderer(
             self.query,
-            self.dialect,
+            self.dialect or model.dialect,
```

Comment:
Does this affect migration v0013? Do we also want to use the model's dialect there as a fallback?

Reply:
This change only impacts runtime and doesn't impact the state.

Reply:
Ah makes sense, since it's just the renderer's dialect that's changed.